docs/controllers/services/examples/http2-with-cert-nginx.yml

```diff
@@ -6,6 +6,7 @@ metadata:
   annotations:
     service.beta.kubernetes.io/do-loadbalancer-protocol: "http2"
     service.beta.kubernetes.io/do-loadbalancer-certificate-id: "c019bbf4-cad2-4884-8a20-410ddad7f54a"
+    # service.beta.kubernetes.io/do-loadbalancer-hostname: "example.com"
```

Comment:
How about putting this in an extra example and leaving the annotation uncommented while putting an explanatory comment right above it, e.g.:

```
annotations:
  service.beta.kubernetes.io/do-loadbalancer-protocol: "http2"
  service.beta.kubernetes.io/do-loadbalancer-certificate-id: "c019bbf4-cad2-4884-8a20-410ddad7f54a"	    
  # "example.com" should be configured to point at the IP address of the DO load-balancer
  service.beta.kubernetes.io/do-loadbalancer-hostname: "example.com"
```